kcbq-connector/quickstart/properties/connector.properties

```diff
@@ -22,18 +22,23 @@ sanitizeTopics=true
 
 autoUpdateSchemas=true
 
-schemaRetriever=com.wepay.kafka.connect.bigquery.schemaregistry.schemaretriever.SchemaRegistrySchemaRetriever
-schemaRegistryLocation=http://localhost:8081
+schemaRetriever=com.wepay.kafka.connect.bigquery.retrieve.IdentitySchemaRetriever
 
 bufferSize=100000
 maxWriteSize=10000
 tableWriteWait=1000
 
+transforms=RegexTransformation
+transforms.RegexTransformation.type=org.apache.kafka.connect.transforms.RegexRouter
+# .* is a placeholder for the actual regex
```

Comment:
Maybe rephrase as something like "A placeholder regex router SMT that does nothing. Replace with relevant transformation SMT"